WordPress/Classes/Utility/PhotonImageURLHelper.h

```diff
@@ -10,7 +10,9 @@
  The source image is resized and the URL is constructed with a
  default 80% quality as a speed/size optimization.
 
- @param size The desired size of the photon image. 
+ @param size The desired "points" size of the photon image. The scale of the screen will be
+        multiplied to this value. If height is set to zero the returned image will have a
+        height proportional to the requested width.
```

Comment:
Just for the record: In this PR just the documentation was updated, since, as it turns out, [the helper was already expecting points size](https://github.com/wordpress-mobile/WordPress-iOS/blob/develop/WordPress/Classes/Utility/PhotonImageURLHelper.m#L28)